skin.css

```diff
@@ -1,13 +1,42 @@
 .highlight {
-    background-color: #ddd;
+	background-color: #ddd;
 }
 
 button.btn {
-  border-radius: 3px;
-  border-style: solid;
-  border-width: 1px;
-  font-weight: normal;
-  line-height: 1.15;
-  text-align: center;
-  vertical-align: middle;
+	margin-left: 5px;
+	padding: 10px 65px;
+	font-size: 18px;
+	line-height: 1.5;
+	font-weight: normal;
+	text-align: center;
+	vertical-align: middle;

```

Comment:
Similar to above, this declaration isn't necessary. It isn't necessary because it only applies to elements which are inline or table-cell - see [here](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align).